### PR TITLE
Implement Wwise tools, and adds Wwise audio project

### LINKED ISSUE #16: Invalid Wwise GitIgnore
**Describe the bug**
53880f1 introduce some issues which, in the current state, cause `main` to no longer be stable: 
* A wild card for *all* meta files was added
It overrides unity's gitignore entry which deliberately includes meta files under /assets/. 
Thus, unity asset settings are now ignored whilst Unity's [docs](https://docs.unity3d.com/Manual/ExternalVersionControlSystemSupport.html) state: 
> When creating new assets, make sure both the asset itself and the associated .meta file is added to version control.

 * Directories that wwise docs specifically say not to include in version control are still included. caches and wsettings are specified in the [documentation](https://www.audiokinetic.com/library/edge/?source=Help&id=using_wwise_with_source_control_system) to not be included.

Additionally, there are 3 files containing GUID's that were changed when i loaded the project locally. 
I can't find any mentions of these online.
* Are these user specific? 
* Should they be ignored?

Screen Shot 2020-12-21 at 2 39 42 pm

I can only seem to find [this](https://forum.unity.com/threads/gitignore-file-for-wwise.446489/) definitive gitignore entry suggestion for wise, but it's four years old. It may be too out dated.

I need some second opinions, i don't know enough about wwise.

**Screenshots**
Screen Shot 2020-12-21 at 2 27 16 pm
Screen Shot 2020-12-21 at 2 23 16 pm

**Development Version**
* Checked out commit: 53880f1
* Branch name: main

@galdchen With all due respect, i kindly ask you to revert 53880f1 from `main` whilst it's still the topmost commit; We should handle wwise intergration in a branch based from `develop` instead.

@atallahhezbor P1 : Main is not stable

### PR DESCRIPTION
**Pull request by Shinkson47**
Closes #16
P1: Wwise integration is a project dependancy

* Implements @galdchen 's wwise integration into this sprint.
	* Adds Wwise audio project data
	* Adds Wwise unity integration tools

* Adds ignores for wwise according to documentation and online advice
* Asserts that assets meta data is still included with accordance to the unity documentation

Tested on two machines with fresh project copies; Everything seems to be as expected, I just want to check that wwise is working correctly between contributors with a larger test sample size.

# Reviewers:
(You'll need the [wwise launcher](https://www.audiokinetic.com/download/))

Checkout `develop-wwise` and check:
* Unity asset meta data works as expected, and new assets's meta files are visible in your git changelists
* wwise launcher detects integration with the `torte` unity project, and shows `modify wwise in project`, and *NOT* `intergrate wwise into project...`
* Unity editor has loaded intergration tools and menus:
` Windows > Wwise Picker`,
`Assets > Wwise`,
` GameObject > Wwise`
 etc

### CODE REVIEW

File: logRunSetup.txt [hunk 1]
```diff
@@ -0,0 +1,5373 @@
+[LicensingClient] ERROR Failed to connect to local IPC
```
Comment (atallahhezbor) [inline logRunSetup.txt:1]: I doubt we need this log file?

**Review by atallahhezbor — COMMENTED** [review]
I can't speak to the majority of these changes, but I do encounter some errors starting the project on this branch. Not sure if that's expected for now

```
Wwise: Cannot open file: Init.bnk
UnityEngine.Debug:LogError(Object)
AkCallbackManager:PostCallbacks() (at Assets/Wwise/Deployment/API/Handwritten/Common/AkCallbackManager.cs:329)
AkSoundEngineController:LateUpdate() (at Assets/Wwise/Deployment/Components/AkSoundEngineController.cs:46)
AkInitializer:LateUpdate() (at Assets/Wwise/Deployment/Components/AkInitializer.cs:87)
```

Also, could the additional setup of the wise launcher as well as anything needed to resolve the above be added to the contributing guidelines?

Lastly, there are two zip files included here, are they necessary / should we be doing anything with them?

File: logRunSetup.txt (hunk 1 above)
Comment (Shinkson47) [reply logRunSetup.txt]: Good shout, i missed that

**Comment by Shinkson47** [comment]
@atallahhezbor 

* Your sound banks were unable to be loaded because none had been generated yet, fb0a3fd fixes that.
* You're absolutely right about those zips and the log; They're unnecessary install remnants. f436690 and 0062e60 cleans those up.
* Lastly, Unity loads and seems to execute just fine without the Wwise launcher or editor. So long as we can validate that integration tools are loaded as expected between collaborators, there's no requirement to have wwise or it's launcher.

 🚀

**Review by atallahhezbor — APPROVED** [review]
LGTM as far as i can tell

**Comment by realQuartzi** [comment]
Everything works fine for me... I have no errors on my side.

**Review by realQuartzi — COMMENTED** [review]
I approve

**Comment by Shinkson47** [comment]
I'll leave `develop-wwise` alive until candidate release is tagged, just in case.

**Pull request merged**